src/rf/apps/core/management/commands/sampledata.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import random
+from datetime import datetime, timedelta
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from apps.core import enums
+from apps.core.models import (Layer, LayerImage, LayerTag,
+                              LayerMeta, UserFavoriteLayer)
+from random_words import RandomNicknames, RandomWords, LoremIpsum
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        rn = RandomNicknames()
+        rw = RandomWords()
+        li = LoremIpsum()
+
+        layers = Layer.objects.all()
+        layer_ids = layers.values_list('id', flat=True)
+
+        self.stdout.write('Deleting records')
+
+        LayerMeta.objects.filter(layer__id__in=layer_ids).delete()
+        LayerImage.objects.filter(layer__id__in=layer_ids).delete()
+        LayerTag.objects.filter(layer__id__in=layer_ids).delete()
+        UserFavoriteLayer.objects.all().delete()
+
+        User.objects.filter(is_superuser=False).delete()
+        layers.delete()
+
+        # These usernames all have one thing in common.
+        usernames = [
+            'dudgeon',
+            'wigeon',
+            'surgeon',
+            'georgian',
+            'gorgeous',
+            'outrageous',
+            'courageous',
+            'george',
+            'gudgeon',
+            'geocaching',
+            'geodesic',
+            'geology',
+            'widgeon',
+            'geologic',
+            'pigeon',
+            'geometry',
+            'geologist',
+        ]
+
+        self.stdout.write('Generating sample data...')
+
+        for username in usernames:
+            u = User()
+            u.username = username
+
+            gender = random.choice(['m', 'f', 'u'])
+
+            u.first_name = rn.random_nick(gender=gender)
+            u.last_name = rn.random_nick(gender=gender)
+            u.is_active = True
+            u.save()
+
+        for user in User.objects.all():
+            for i in range(random.randint(3, 10)):
+                layer = Layer()
+                layer.user = user
+
+                layer.name = li.get_sentence()
+                layer.description = li.get_sentences(2)
+                layer.organization = ' '.join(rw.random_words(count=5))
+
+                layer.is_public = random.random() <= 0.80
```

Comment:
Making 20% of them private is nice. If we make them private by default and the user needs to change them to public, then it'll probably be the other way around.